sync/config/triggers.yaml

```diff
@@ -30,6 +30,48 @@ archive: https://github.com/tektoncd/triggers/tags
 #       header: <dict>         # optional, no header added if not set
 #         See https://www.docsy.dev/docs/adding-content/navigation/#section-menu
 tags:
+- name: v0.13.0
```

Comment:
NIT: could you keep the list indentation consistent with the rest of the config?

Reply:
Sure thing! But I think this is consistent? The rest of the config is not indented either.

Reply:
Oh, never mind, it looks like it's the way github displays it that made it look different? Sorry about that...